pkg/controller/podgroup.go

```diff
@@ -374,10 +374,11 @@ func addResources(minResources corev1.ResourceList, resources corev1.ResourceReq
 		}
 	}
 	for name, quantity := range merged {
+		addition := resource.MustParse(strconv.Itoa(int(quantity.Value()) * replicas))
```

Comment:
Ah, I didn't realize that Quantity doesn't have a multiplication operation.

This is an odd way of calculating it though. Have you verified if this works for something like `1m`?

If not, maybe we can revert and follow up with a PR upstream to add multiplication.

Reply:
FWIW, in kueue we convert everything to int64 before multiplying and assume, using `MilliValue` for CPU.

Reply:
> Have you verified if this works for something like 1m?

I verified that situation right now. This way doesn't work well :(

Reply:
> FWIW, in kueue we convert everything to int64 before multiplying, using MilliValue for CPU.

I verified  the `resource.MustParse(strconv.Itoa(int(quantity.MilliValue()) * replicas))` works well.

Reply:
However, as you say, this way looks odd.

Reply:
are you sure it gives you the right value?

also, it might not work well with memory.

Reply:
> are you sure it gives you the right value?
>
> also, it might not work well with memory.

If we use `MilliValue` for CPU and Memory, it seems to work well. However, for GPU, it doesn't work well.

Reply:
However, I’m not sure the above conversion works well in all situations.

So, It might be better to revert this change.

Reply:
+1

we should follow up with a multiplication method upstream. It will become increasingly useful for batch operators.

Reply:
I will create an issue in k/k tomorrow or next week.

Reply:
reverted.